### ISSUE TITLE
Laden des Rezepts fehlgeschlagen (GER) / opening of recipe fails

### ISSUE THREAD

**Issue by SnorreQWERTY** [mrzapp/nextcloud-cookbook#283]
**Description**
message like in header

**Reproduction**
- click on recipe ....
- error message opens

Firefox on Linux Mint

recipe worked correct with Version 18. With version 19 there is the problem. Error occurs also with the new update to 0.7.6

**Comment by SnorreQWERTY** [comment]
additional information:

I found two folders ...
a) the original folder "recipes" and 
b) a new folder "rezepte". 

- I copied my only recipe into the new folder.
- by clicking on "Kochbuch" in the menu I get two recipes. Header is: "alle Rezepte"
- the left one fails (the old one?!?!?) .... the right one works

renaming the folder triggers the problem in "all" no-english versions

the recipes should moved into the new folder and temporarily there should be any comparing / merging if there are two identically recipes.


Feature-Request:
-----------------------
This plugin / all plugins with representation in the file-system should use no names which are colliding with MY alphabetical order of MY personal folders with data. ==> Use any sign as "first letter" PLUS underline which brings these folders to the last position.

Optimum will be, that ALL plugins use the same schema of naming.

actual name conventions will bring rezepte to "R" of alphabetical order. Opening the folder makes no sense. There is a JSON file. Reading the json file is not for humans and will not open the cookbook automatically.
